src/avatar.jsx

```diff
@@ -86,6 +83,10 @@ const Avatar = React.createClass({
     this.setState({muiTheme: newMuiTheme});
   },
 
+  getTheme() {
```

Comment:
Yeah I think we should stop using the function. I haven't see the `getTheme` function pattern around in the library much but if it's around, we should move towards not doing it.

Personally, even if we need a function to help with codeflow/logic I'd rather put the function at the top of the module rather as a member of the React component. The more functions we have of components, the more likely someone will read source and abuse them.

Actually, this is one of the reasons why I prefer using functional components when possible over classes because it just takes the choice away.

In addition, this particular issue is why I was hoping we could figure out something like #2509. A lot of times when we do have the theme, we do have to traverse down a lot of nested properties to get what we need. I'm still not sure best approach but I'm not entirely convinced our current one is the best one.